contracts/src/rollup/LineaRollup.sol

```diff
@@ -17,7 +17,7 @@ contract LineaRollup is AccessControlUpgradeable, ZkEvmV2, L1MessageService, Per
   using EfficientLeftRightKeccak for *;
 
   /// @notice This is the ABI version and not the reinitialize version.
-  string public constant CONTRACT_VERSION = "6.0";
+  string public constant CONTRACT_VERSION = "6.1";
```

Comment:
given the nature of this change, this is a breaking change on the API, I would go for `7.0`

Reply:
done